### PR TITLE
Episode 6, Part 2: show notes, feed

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
cc @sockeqwe, @android10

### CODE REVIEW

File: feed.rss [hunk 1]
```diff
@@ -356,5 +356,54 @@
 			]]>
 		</itunes:summary>
 	</item>
+	<item>
+		<guid>https://artemzin.com/static/thecontext/episodes/The.Context.episode.6.part2.mp3</guid>
+		<title>Episode 6, Part2: Continuous Integration (CI) and Continuous Delivery (CD) with Fernando Cejas from SoundCloud (&amp; EVERYTHING &amp; LIFE)</title>
```
Comment (artem-zinnatullin) [inline feed.rss]: TODO: update build time in the feed

File: feed.rss [hunk 2]
```diff
@@ -356,5 +356,54 @@
...
+				<li>Artem Zinnatullin <a href="https://twitter.com/artem_zin">@artem_zin</a>, <a href="http://artemzin.com">personal blog</a>, <a href="https://github.com/artem-zinnatullin">GitHub</a></li>
+				<li>Hannes Dorfmann <a href="https://twitter.com/sockeqwe">@sockeqwe</a>, <a href="http://hannesdorfmann.com">personal blog</a>, <a href="https://github.com/sockeqwe">GitHub</a></li>
+			</ul>
+
+			<h4>Additional links:</h4>
+
+			<ul>
+				<li><a href="https://speakerdeck.com/android10/it-is-about-philosophy-culture-of-a-good-programmer-second-edition">Slides from Fernando's talk at MobileOptimized'2016 </a></li>
+				<li><a href="https://github.com/android10/Android-CleanArchitecture">Fernando's Clean Architecture</a> sample on github</li>
+				<li><a href="https://todo.hannes">TODO Hannes LINK HERE</a></li>
```
Comment (artem-zinnatullin) [inline feed.rss]: TODO: @sockeqwe

File: show_notes/Episode_6_part2.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+#### Hosts:
+
+  - Artem Zinnatullin [@artem_zin](https://twitter.com/artem_zin), [personal blog](http://artemzin.com), [GitHub](https://github.com/artem-zinnatullin)
+  - Hannes Dorfmann [@sockeqwe](https://twitter.com/sockeqwe), [personal blog](http://hannesdorfmann.com), [GitHub](https://github.com/sockeqwe)
+
+#### Additional links:
+
+ - [Slides from Fernando's talk at MobileOptimized'2016 ](https://speakerdeck.com/android10/it-is-about-philosophy-culture-of-a-good-programmer-second-edition)
+ - [Fernando's Clean Architecture](https://github.com/android10/Android-CleanArchitecture) sample on github
+ - [TODO Hannes LINK HERE](https://todo.hannes)
```
Comment (artem-zinnatullin) [inline show_notes/Episode_6_part2.md]: TODO: @sockeqwe

**Comment by sockeqwe** [comment]
Thanks man!

But is the file already uploaded? I get a  Http `404`. Maybe the link is wrong: https://artemzin.com/static/thecontext/episodes/The.Context.episode.6.part2.mp3

**Comment by artem-zinnatullin** [comment]
No, not uploaded yet!

**Comment by sockeqwe** [comment]
Ok, I have added the missing links and updated the date - time values.

Looks good to me, can be released now. Just ping me when you have uploaded the `mp3` file, then I will press the merge button (you could merge this pr directly too)

**Comment by android10** [comment]
LGTM! 👍

File: feed.rss (hunk 1 above)
Comment (artem-zinnatullin) [inline feed.rss]: done!

**Comment by artem-zinnatullin** [comment]
Boom!

**Pull request merged**